### ISSUE TITLE
Can we have the option to view in "List View"

### ISSUE THREAD

**Issue by DavidAyliffe** [ganeshrvel/openmtp#369]
Can we have the option to view in "List View" please

**Comment by ganeshrvel** [comment]
Hi, OpenMTP author here. Glad to hear you found the application working for you! I'm already working on a lighter version of the app with many new features. Stay tuned!

**Comment by ganeshrvel** [comment]
You may find the option in settings, toggle off Grid view.

**Comment by bitboxer** [comment]
Sadly to say this is super confusing. Searched 10 min and then searched issues to see if someone has already opened one😅

**Comment by huyi122** [comment]
I can't complain, this free software is really great, thanks for the contribution. but toggle "grid view" can be placed on the tool-bar for better user exp.

**Comment by inspiredearth** [comment]
Agreed. Although it's nice to have this option in the Settings as a default preference, it would be great to have it as a toggle in the toolbar.

**Comment by djlongy** [comment]
Can we makke the columns resizable so we can easily see the full length filenames?
